config/locales/en.yml

```diff
@@ -52,3 +52,8 @@ en:
   time:
     formats:
       day_and_date: "%A, %v"
+      mm_dd_yyyy: "%m-%d-%Y"
```

Comment:
Can we give this one an actual name, instead of one that describes the date format? While this app will probably not need to be localized, in the spirt of leaving that option open, we should use names that make sense in other languages. For example, if we were to deploy this in Europe, this format would probably look like:

```
de:
  time:
    formats:
      mm_dd_yyyy: "%d-%m-%Y"
```

in which case the label is less useful.  We could use something like "standard" or "default" for the label, or anything else that makes sense to you.